web/app.php

```diff
@@ -1,15 +1,6 @@
 <?php
 
-use Symfony\Component\HttpFoundation\Request;
+putenv('APP_ENV=dev');
+putenv('APP_DEBUG=1');
 
-require __DIR__ . '/../vendor/autoload.php';
-
-$kernel = new AppKernel('prod', false);
-//$kernel = new CacheKernel($kernel);
-
-// When using the HttpCache, you need to call the method in your front controller instead of relying on the configuration parameter
-//Request::enableHttpMethodParameterOverride();
-$request = Request::createFromGlobals();
-$response = $kernel->handle($request);
-$response->send();
-$kernel->terminate($request, $response);
+require __DIR__ . '/../public/index.php';
```

Comment:
this is the trick that keeps BC, keeping the app.php file but internally using the new index one.
small BC break here is for users who have altered their app.php file to use the AppCache, now they'll need to alter the index.php to use CacheKernel.
see https://symfony.com/doc/4.4/http_cache.html#symfony-reverse-proxy

I think it's acceptable as I don't think users do that, maybe I'm wrong